migration/v0.34/apply/30-delete-obsolete-ns.sh

```diff
@@ -6,16 +6,23 @@ ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
 source "${ROOT}/scripts/migration/lib.sh"
 
 run() {
+  local cluster="${2:-both}"
```

Comment:
Use the environment variable instead of an argument if you go with an environment variable.